backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/webservice/schema/CardMutationService.kt

```diff
@@ -99,8 +104,12 @@ class CardMutationService {
         return CardActivationResultModel(ActivationState.success, encodedTotpSecret)
     }
 
-    private fun validateNewCard(card: CardGenerationModel): Boolean {
+    private fun validateCodeType(card: CardGenerationModel): Boolean {
         return (card.codeType == CodeType.STATIC && card.activationSecretBase64 == null) ||
             (card.codeType == CodeType.DYNAMIC && card.activationSecretBase64 != null)
     }
+
+    private fun validateCardHash(cardHash: ByteArray): Boolean {
+        return cardHash.size != 32
```

Comment:
I think the function validateCardHash should not return true, if card hash is invalid.

Imo, it would be better to name the function `isCardHashValid` to avoid any confusion. (same for validateCodeType)